CONTRIBUTING.md

```diff
@@ -86,53 +81,21 @@ that to be accepted into the repository, the pull request must [pass all tests](
-  There are some tests that appear in all repositories.
-  These tests can be found in the following folders inside of your target build directory:
-
-  (These instructions are for Linux)
-* In the `demos` directory, run:
-
->        vkcube
->        vkcube --validate
->        smoke
->        smoke --validate
->        vulkaninfo
-
-* In the `tests` directory, run:
-
->        run_all_tests.sh
-
-* On Windows, a quick sanity check can be run from inside Visual Studio -- just run the `vk_layer_validation_tests` project,
-or you can run `run_all_tests.ps1` from a PowerShell window
-
+  The tests can be found in the test directory inside of your target build directory
 * Note that some tests may fail with known issues or driver-specific problems.
   The idea here is that your changes should not change the test results, unless that was the intent of your changes.
 * Run tests that explicitly exercise your changes.
 * Feel free to subject your code changes to other tests as well!
 
 
-#### **Special Considerations for Validation Layers**
-* **Validation Tests**  If you are submitting a change that adds a new validation check, you should also construct a "negative" test function.
-The negative test function purposely violates the validation rule that the new validation check is looking for.
-The test should cause your new validation check to identify the violation and issue a validation error report.
-And finally, the test should check that the validation error report is generated and consider the test as "passing"
-if the report is received.  Otherwise, the test should indicate "failure".
-This new test should be added to the validation layer test program in the `tests` directory and contributed
-at the same time as the new validation check itself, along with appropriate updates to `layers\vk_validation_error_database.txt`.
-There are many existing validation tests in this directory that can be used as a starting point.
-* **Validation Checks**  The majority of validation checks are carried out by the Core Validation layer. In general, this layer
-contains checks that require some amount of application state to carry out. In contrast, the parameter validation layer contains
-checks that require (mostly) no state at all. Please inquire if you are unsure of the location for your contribution. The other
-layers (threading, object_tracker, unique_objects) are more special-purpose and are mostly code-generated from the specification.
-
 ### **Contributor License Agreement (CLA)**
 
 You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request
 or other contribution to GitHub.
 
 ### **License and Copyrights**
 
-All contributions made to the Vulkan-ValidationLayers repository are Khronos branded and as such,
+All contributions made to the VulkanTools are LunarG branded and as such,
 any new files need to have the Khronos license (Apache 2.0 style) and copyright included.
```

Comment:
"Khronos license" => "LunarG license", or not?

Reply:
It's not a lunarG license either. I would simply say any new files need to have the Apache 2.0 license.